### DISCUSSION TITLE
Rest API set(update) a column of a field to Null

### DISCUSSION THREAD

**Discussion by imcouri** [supabase/supabase#3066] [Questions]
Hello,
I'm creating an app where I need to remove a specific column from a row and I need to set it to null. I'm using PHP therefore I'm using the API instead of a client. It seems to me that supabase ignores null fields in PATCH requests since it neither updates the value nor returns an error.
Is there a way that I can set a column of a row to null?
Thank you

**Comment by steve-chavez** [comment]
How are you doing the request? Setting to null works fine with plain curl:

```bash
curl  '<url>/projects?id=eq.1334'
[{"id":1334,"name":"blabla","client_id":null,"earnings":100}]  

curl -X PATCH -d '{"earnings": null}' -H "Content-Type: application/json" '<url>/projects?id=eq.1334'

curl  '<url>/projects?id=eq.1334'
[{"id":1334,"name":"blabla","client_id":null,"earnings":null}]  
```
